doc/en/example/reportingdemo.rst

```diff
@@ -9,9 +9,9 @@ and how ``pytest`` presents things (unfortunately
 not showing the nice colors here in the HTML that you
 get on the terminal - we are working on that)::
 
-    $ pytest failure_demo.py
+    assertion $ pytest failure_demo.py
```

Comment:
I only noticed the problem after running regendoc again because it wouldn't find `failure_demo.py` anymore and remove basically all the example failures below the text. :grin: